src/components/shared/button/ButtonComponent.tsx

```diff
@@ -0,0 +1,9 @@
+import Button from "@mui/material/Button";
+
+interface ButtonProps {
+  title: string;
+}
+
+export const ButtonComponent = ({ title }: ButtonProps) => {
+  return <Button variant="contained">{title}</Button>;
```

Comment:
Make this more reusable. What if I want to reuse it and pick a different variant?

Reply:
Removed variant. But in the project almost every button  looks like this variant contained.